### DISCUSSION TITLE
SFlix Download fail

### DISCUSSION THREAD

**Discussion by Avantist** [aclap-dev/video-downloadhelper#565] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://sflixtv.to/movie/stockholm-bloodbath-ywwpz/1-1",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 00000181b0dfa180] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 00000181b0b00600] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 00000181b0bc3f00] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[aac_adtstoasc @ 00000181b0448cc0] Error parsing ADTS frame header!\r\n[mp4 @ 00000181b0474880] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 00000181b0c4b040] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 00000181b044f240] Error muxing a packet\r\n",
    "id": "coapp_failure_df3e0af0-ae30-4504-9b9a-87c9e069b690",
    "downloadable_id": "downloadable_4145608905731117"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\glitc\\Downloads",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 5
}

**Comment by paulrouget** [comment]
Triage: ff-failure. Invalid data found when processing input

**Comment by Mattmjs** [comment]
I'm still on Version 8.2.2.8 (stable) and it appears to download, the speed looks to be throttled a lot.
